src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -392,13 +395,31 @@ void Profile::_FinalizeInheritance()
             }
         }
     }
+    if (auto fontInfoImpl = get_self<FontConfig>(_FontInfo))

```

Comment:
nit: you don't actually need `if` here, right? `_FontInfo` can't be `nullptr`

Reply:
It's there for consistency with the `_DefaultAppearance` check above (which technically also cannot be nullptr, but I feel like it's a 'just in case' kind of thing)

Reply:
Actually if you call `get_self` on a nullptr your application will just straight up crash in almost all cases.